sympy/integrals/tests/test_integrals.py

```diff
@@ -438,14 +438,13 @@ def test_evalf_issue_939():
         integrate(1/(x**5 + 1), (x, 2, 4)), chop=True) == '0.0144361088886740'
 
 
-@XFAIL
 def test_failing_integrals():
```

Comment:
this could now be named "def test_double_integrals():" and strike the words "not implemented" in the line below.

Reply:
done. I named the first assertion as double integral